### PR TITLE
Deny for revision

### PR DESCRIPTION
**Pull request by sagilaufer1992**
add another policy directory, for denying every deployment not from master or main revisions

### CODE REVIEW

**Comment by avnerenv0** [comment]
since this is a public repo please add a README about what this folder is about, even it's very short.

Also I don't see the need to split the rego into three files but that's not very important

File: deny-not-main-or-master-revision/allow.rego [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package env0
+
+# METADATA
+# title: allow by default
```
Comment (avnerenv0) [inline deny-not-main-or-master-revision/allow.rego]: `title` is not needed or used here.

File: deny-not-main-or-master-revision/deny.rego [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package env0
+
+# METADATA
+# title: deny for non-main revisions
+# description: deny if deployment revision is not "main" or "master"
+deny[format(rego.metadata.rule())] {
+    allowed_revisions = { "main", "master" }
+	not allowed_revisions[input.deploymentRequest.revision]
```
Comment (avnerenv0) [inline deny-not-main-or-master-revision/deny.rego:7]: I just hate rego what kind of evil syntax is this

Comment (sagilaufer1992) [reply deny-not-main-or-master-revision/deny.rego:7]: that's how we test if a value is part of the set.
the not is for negation

**Comment by sagilaufer1992** [comment]
> since this is a public repo please add a README about what this folder is about, even it's very short.
> 
> Also I don't see the need to split the rego into three files but that's not very important

we don't have a readme.md for any other repo ATM. we should open an issue, to add for the rest as well probably, but I don't think it should block us from adding this one as it's a simple policy, pretty much like the others

File: deny-not-main-or-master-revision/allow.rego (hunk 1 above)
Comment (sagilaufer1992) [reply deny-not-main-or-master-revision/allow.rego]: [acea3ec](https://github.com/env0/approval-policies/pull/5/commits/acea3ec9f19baac981f37c75a5190d80c09581df)

File: deny-not-main-or-master-revision/env0.rego [hunk 3]
```diff
@@ -0,0 +1,3 @@
+package env0
+
+import data.env0
```
Comment (ItamarMalka) [inline deny-not-main-or-master-revision/env0.rego:3]: just a knowledge question what is this import? 🤔

**Review by ItamarMalka — APPROVED** [review]
nice rule, LGTM 👍 
Would be nice to have a Readme.md of a couple lines describing how it is used an why

File: deny-not-main-or-master-revision/env0.rego (hunk 3 above)
Comment (sagilaufer1992) [reply deny-not-main-or-master-revision/env0.rego:3]: @tomer-landesman would like your input here, I copy-pasted that

**Comment by sagilaufer1992** [comment]
@ItamarMalka added a readme [79b9bb3](https://github.com/env0/approval-policies/pull/5/commits/79b9bb390473c30d9de89a8722d4f70105db0039)

**Pull request merged**